
string  task_ref      # name or typee which can be used as a lookup into a zoo registry

string  task_ver      # optional version (if not specified, defaults to "latest")

string  params_json   # optional JSON string containing config parameters

